src/integration-test/tests/citizen/defence/pages/statement-of-means/carer.ts

```diff
@@ -0,0 +1,25 @@
+import I = CodeceptJS.I
+
+const I: I = actor()
+
+const fields = {
+  yes: 'input[id=optionyes]',
```

Comment:
rename radioButtons to pose the question e.g isCarer, notCarer etc
makes more meaningful pageObject

Reply:
Because of the way the macros are written (from an external project) I can do `carerYes` and `carerNo` and so on for the other pages. Would this be okay?

Reply:
That would be fine